server/service/orbit_client.go

```diff
@@ -338,3 +338,15 @@ func OrbitRetryInterval() time.Duration {
 	}
 	return constant.OrbitEnrollRetrySleep
 }
+
+// SetOrUpdateDiskEncryptionPayload sends a request to the server to set or update the disk
+// encryption keys and result of the encryption process
+func (oc *OrbitClient) SetOrUpdateDiskEncryptionPayload(diskEncryptionStatus fleet.OrbitHostDiskEncryptionKeyPayload) error {
```

Comment:
As mentioned, I'd suggest `SetOrUpdateDiskEncryptionKey` as method name, to match the `Service` method that implements the endpoint.

Reply:
Done - see [here](https://github.com/fleetdm/fleet/pull/14022/files#diff-6697275ba79dc39a42a714475e32a58103bbedf3f3de67b7183f979276049606R352)